Integrations/Flashpoint/Flashpoint.yml

```diff
@@ -148,28 +157,29 @@ script:
       type: string
     - contextPath: File.Flashpoint.href
       description: List of indicators reference.
-      type: unknown
     - contextPath: File.Malicious.Description
       description: Description of malicious file.
       type: string
     - contextPath: File.Malicious.Vendor
       description: Vendor of malicious file.
       type: string
-    - contextPath: File.MD5
+    - contextPath: File.md5
```

Comment:
File.MD5 was the correct way, the context standards should be according to https://github.com/demisto/content/tree/master/docs/context_standards.

You need to change the outputs in the code from `sha1` to `SHA256`, `md5` to `MD5`, etc